woocommerce/woocommerce
# [Settings] Add layout areas and styles

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Partially addresses https://github.com/woocommerce/woocommerce/issues/52275

<img width="867" alt="image" src="https://github.com/user-attachments/assets/effff3e6-5c8b-4e5c-8fac-d00182c7efb3">




### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Enable the Product Editor since it contains code we'll need. WooCommerce > Settings > Advanced > Features > Try the new Product Editor.
2. Install Gutenberg and turn on experiments Gutenberg > Experiments > Custom dataviews
4. Turn on the `settings` and `product-data-views` feature flags. Tools > WCA Test Helper > Features
5. WooCommerce > Settings
6. See the sidebar placeholder content render along with site hub.
7. Make sure other admin screens still function



### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

Adds layout to Settings, which is unreleased.

</details>
